drivers/generic/cpssEnabler/linuxNoKernelModule/drivers/ethDriver.c

```diff
@@ -3845,7 +3870,11 @@ static int mvppnd_pci_probe(struct pci_dev *pdev,
 	}
 
 	/* We want 32 bit address space */
+#if LINUX_VERSION_CODE <= KERNEL_VERSION(5,17,0)
```

Comment:
No need for the #if. pci_set_dma_mask() wraps around dma_set_mask() since 3.10.70, and we do not support earlier kernels. Just call dma_set_mask() without #if .

Reply:
Thanks elad. will call dma_set_mask() without #if